src/components/dictionaryConcepts/components/MappingModal.jsx

```diff
@@ -7,19 +7,24 @@ import {
 import InternalMapping from './InternalMapping';
 import ExternalMapping from './ExternalMapping';
 
+const externalMapping = 'External Mapping';
```

Comment:
Can you point me to the convention you are looking at for names of constants?

Reply:
I am using Camel case for names of constants.

Reply:
Can you point me to where you did read that?

Reply:
https://en.wikipedia.org/wiki/Camel_case

Reply:
Did you understand my question?

Reply:
I most likely did not understand, kindly suggest a better alternative. I am thinking it may be that the first letter should be uppercase, though some people say it should start with lower case. Is it what I am thinking?

Reply:
Have you ever read any document about variable naming conventions?

Reply:
I have read a few which I cannot recall or find now but this is one of them. https://github.com/andela/bestpractices/wiki/JavaScript-Style-Guide

Reply:
We should use camel case to name constants, one link in the Andela doc I put in a previous message included the Airbnb style guideline which we are using for this project. [Here](https://github.com/airbnb/javascript#naming-conventions) is a section talking about naming conventions for this style guideline (Airbnb).

Reply:
Can you copy and paste the wording from that convention document about names for constants?

Reply:
```
23.2 Use camelCase when naming objects, functions, and instances. eslint: camelcase

// bad
const OBJEcttsssss = {};
const this_is_my_object = {};
function c() {}

// good
const thisIsMyObject = {};
function thisIsMyFunction() {}```

Reply:
I do not see constants in that sentence.

Reply:
I am now a bit stuck, I totally do not understand the question, kindly let me know where I am going wrong.

Reply:
Can you do some 5 minutes googling, or ask your team mates about any recommended naming convention for constants?

Reply:
Ok @dkayiwa, I will do just that.

Reply:
I have made changes to the constant variable naming, please take a look into this PR again.